Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Collections/BaseCollection.cs

```diff
@@ -0,0 +1,134 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.UX.Collections
+{
+    public abstract class BaseCollection : MonoBehaviour
```

Comment:
Should we have a collection interface?

Reply:
What are you thinking would be included in it vs the base class? LayoutChildren()?

Reply:
Anything that defines a collection.